bot/cogs/antimalware.py

```diff
@@ -1,11 +1,12 @@
 import logging
 
-from discord import Message, NotFound
+from discord import Embed, Message
 from discord.ext.commands import Bot, Cog
 
 from bot.constants import AntiMalware as AntiMalwareConfig, Channels
 
 log = logging.getLogger(__name__)
+PASTE_URL = "https://paste.pythondiscord.com/"
```

Comment:
https://paste.pydis.com/ is shorter (which is important for a paste service)

Reply:
I'd rather it use the value [defined in the constants](https://github.com/python-discord/bot/blob/master/bot/constants.py#L449) so this doesn't have to be hardcoded.